napari/layers/_image_layer/model.py

```diff
@@ -187,17 +187,10 @@ def _slice_image(self):
         """Determines the slice of image from the indices."""
 
         indices = list(self.indices)
-
-        for dim in range(len(indices)):
-            max_dim_index = self.image.shape[dim] - 1
-
-            try:
-                if indices[dim] > max_dim_index:
-                    indices[dim] = max_dim_index
-            except TypeError:
-                pass
-
+        indices[:-2] = np.clip(indices[:-2], 0,
+                               np.subtract(self.shape[:-2], 1))
         self._image_view = np.asarray(self.image[tuple(indices)])
+        self._image_thumbnail = self._image_view
```

Comment:
I find this to be a surprising bit of code! Why did you alias this?

Reply:
I needed to alias this in the `Image` layer so that it could be defined differently in the `Pyramid` layer (which subclasses the `Image` layer). While in the `Image` layer it makes sense for this to be the same as the `_image_view` in the `Pyramid` layer you want this to correspond to the view for the lowest resolution of the pyramid